casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -22,15 +27,36 @@ final case class Genesis(
 )
 
 object Genesis {
+  val genesisPubKey = PublicKey(Array[Byte]())
 
-  val NonNegativeMergeableTagName: Par = {
-    val rand = Tools.unforgeableNameRng(
-      StandardDeploys.nonNegativeNumberPubKey,
-      StandardDeploys.nonNegativeNumberTimestamp
-    )
-    import coop.rchain.models.rholang.implicits._
+  def NonNegativeMergeableTagName(
```

Comment:
Functions should be start with lower case, only object constant should start with capital letter.